module1/build tools/multi-project/core/build.gradle

```diff
@@ -0,0 +1,15 @@
+plugins {
+    id 'java'
+}
+
+group 'com.epam'
+version '1.0'
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    testCompile group: 'junit', name: 'junit', version: '4.12'
+    compile files('C:\\Users\\kirill\\Desktop\\lab\\module1\\build tools\\jar\\utils-1.3.5.jar')
```

Comment:
@pilinoga,  If I decide to test your project on my local environment, my absolute path will be different. Use relative path to 'utils-1.3.5.jar' instead.

Reply:
> @pilinoga, If I decide to test your project on my local environment, my absolute path will be different. Use relative path to 'utils-1.3.5.jar' instead.

Used relative path to utils-1.3.5.jar.